format/FlightSql.proto

```diff
@@ -89,6 +90,31 @@ enum SqlInfo {
    */
   FLIGHT_SQL_SERVER_READ_ONLY = 3;
 
+  /*
+   * Retrieves a boolean value indicating whether the Flight SQL Server supports executing Substrait plans.
```

Comment:
So "Flight SQL" is generic enough that it might support things other than actual SQL?

Reply:
Yes, the naming is unfortunate, but there's no real reason why we need to be tied to SQL specifically, and no reason why many of the existing concepts can't map to Substrait.